docs/user/cassandra_datastore.rst

```diff
@@ -83,6 +83,27 @@ Using the Cassandra Data Store in Geoserver
 
 See :doc:`./geoserver`.
 
+
+Cassandra DataStore Command Line Tools
+--------------------------------------
+
+GeoMesa provides a set of command line tools for working with a Cassandra data store.
+To get a description on how to use the tools, type ``geomesa-cassandra``.
```

Comment:
can you reference where this script lives? I assume in the cassandra dist, but it wouldn't be on the classpath automatically right?